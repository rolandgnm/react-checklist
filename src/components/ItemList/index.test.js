import React from 'react';
import ListItem from './index';
import renderer from 'react-test-renderer';

describe('<ListItem />', () => {

	it('should render a list', () => {
		const tree = renderer.create(
			<ListItem items={[{description: 'my first item'}]}/>
		).toJSON();

		expect(tree).toMatchSnapshot();

	});

});