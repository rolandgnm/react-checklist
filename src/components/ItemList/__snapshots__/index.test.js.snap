// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListItem /> should render a list 1`] = `
<ul>
  <li
    onClick={[Function]}
  >
    my first item
  </li>
</ul>
`;
